app/src/main/java/com/fmpdroid/moneybox/dto/MoneyBoxDto.java

```diff
@@ -0,0 +1,16 @@
+package com.fmpdroid.moneybox.dto;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+public @NoArgsConstructor class MoneyBoxDto {
+
+    private @Getter @Setter String Title;
+    private @Getter @Setter String Description;
+    private @Getter @Setter float TargetAmount;
+    private @Getter @Setter String TargetDate;
+    private @Getter @Setter boolean AllowReminder;
+    private @Getter @Setter String DateCreated;
+
+}
```

Comment:
1. fields should start small then go camelcase per word or if deemed necessary. You can use "Right click field>Refactor>Rename..." in Android Studio function to recursively refactor the method names.
2. remove getter and setter annotations per field. instead, use @Data
3. Annotations should be above the targeted line

```suggestion
@NoArgsConstructor
@Data
public class MoneyBoxDto {

    private String title;
    private String description;
    private float targetAmount;
    private String targetDate;
    privateboolean allowReminder;
    private String dateCreated;

}
```